docs/tendermint-core/using-tendermint.md

```diff
@@ -42,7 +42,7 @@ definition](https://github.com/tendermint/tendermint/blob/master/types/genesis.g
 - `genesis_time`: Official time of blockchain start.
 - `chain_id`: ID of the blockchain. This must be unique for
   every blockchain. If your testnet blockchains do not have unique
-  chain IDs, you will have a bad time. The ChainID must be less than 50 bytes.
+  chain IDs, you will have a bad time. The ChainID must be less than 50 symbols.
```

Comment:
Do we allow unicode? Why is it symbols not bytes

Reply:
200 bytes (or 50 UTF-8 symbols 4 bytes each or 100 UTF-8 symbols 2 bytes each)